diff --git a/include/perfetto/ext/base/sys_types.h b/include/perfetto/ext/base/sys_types.h
--- a/include/perfetto/ext/base/sys_types.h
+++ b/include/perfetto/ext/base/sys_types.h
@@ -26,9 +26,10 @@

 #if PERFETTO_BUILDFLAG(PERFETTO_OS_WIN)

+using uid_t = unsigned int;
+
 #if !PERFETTO_BUILDFLAG(PERFETTO_COMPILER_GCC)
 // MinGW has these. clang-cl and MSVC, which use just the Windows SDK, don't.
-using uid_t = unsigned int;
 using pid_t = int;
 #endif  // !GCC

